server/service/integration_enterprise_test.go

```diff
@@ -3689,6 +3687,62 @@ func (s *integrationEnterpriseTestSuite) TestMDMAppleOSUpdates() {
 	s.assertAppleOSUpdatesDeclaration(nil, mdm.FleetIPadOSUpdatesProfileName, nil)
 }
 
+// Skipping admin-created users because we don't have email fully set up in integration tests
```

Comment:
Is the `MAIL_TEST` thing useful?

Reply:
That just gets used for determining whether to skip tests entirely from what I'm seen. Mailer configuration happens elsewhere so integration_core/enterprise_test have set dependencies. I have some tests at the service level with a mocked mailer to make sure things behave as expected.

Reply:
Yeah, that would absolutely be useful here. Would just need to go in a different test.

If I have to touch the code at all for subsequent work I'll likely add that.